src/systemTest/java/com/flightstats/hub/system/functional/ChannelLatestUpdatedStateTest.java

```diff
@@ -0,0 +1,161 @@
+    @Inject
+    private ChannelItemRetriever channelItemRetriever;
+    @Inject
+    private ChannelItemCreator channelItemCreator;
+    @Inject
+    private InternalZookeeperService zookeeperService;
+
+    @BeforeEach
+    void setup() {
+        channelName = randomAlphaNumeric(10);
+    }
+
+    @AfterEach
+    void cleanup() {
+        channelConfigService.delete(channelName);
+    }
+
+    @Test
+    void channelWithNoItem_hasStateValueOfEpochNone_and_hasNoChannelLatest() {
+        channelConfigService.createWithDefaults(channelName);
+
+        assertChannelLatestUrlIsBlank();
+        assertZookeeperLatestUpdatedItemPathEquals(NO_CHANNEL_LATEST_ZK_VALUE);
+    }
+
+    @Test
+    void channelWithItemsInSpoke_returnsLatestInZkAndEndpoint() {
+        channelConfigService.createWithDefaults(channelName);
+        List<ChannelItemWithBody> channelItems = channelItemCreator.addItems(channelName, "data", 3);
+        ChannelItemWithBody expectedLastItem = channelItems.stream()
+                .min((other, item) -> item.getDateTime().compareTo(other.getDateTime()))
+                .orElseThrow(AssertionError::new);
+
+        assertChannelLatestUrlEquals(expectedLastItem.getItemUrl());
+        assertZookeeperLatestUpdatedItemPathEquals(expectedLastItem.getZookeeperItemPath());
+    }
+
+    @Test
+    void historicalChannelWithItemsInTheMutabilityWindow_showsUpAsHavingNoZkLatestOrChannelLatest() {
+        DateTime now = DateTime.now();
+        ChannelConfig config = ChannelConfig.builder()
+                .name(channelName)
+                .mutableTime(now)
+                .build();
+        channelConfigService.create(config);
+
+        channelItemCreator.addHistoricalItems(channelName,
+                Arrays.asList(now.minusMinutes(20), now.minusMinutes(21)));
+
+        assertChannelLatestUrlIsBlank();
+        assertZookeeperLatestUpdatedItemPathEquals(NO_CHANNEL_LATEST_ZK_VALUE);
+    }
+
+    @Test
+    void channelWithItemsOutsideSpokeWindow_hasStateEqualToChannelLatest() {
+        DateTime now = DateTime.now();
+        ChannelConfig config = ChannelConfig.builder()
+                .name(channelName)
+                .mutableTime(now)
+                .build();
+        channelConfigService.create(config);
+
+        SortedSet<ChannelItemWithBody> channelItems = channelItemCreator.addHistoricalItems(channelName,
+                Arrays.asList(now.minusMinutes(9).minusSeconds(45), now.minusMinutes(9).minusSeconds(44)));
+        ChannelItemWithBody mostRecentItem = channelItems.last();
+
+        ChannelConfig updatedConfig = ChannelConfig.builder()
+                .name(channelName)
+                .ttlDays(5)
+                .build();
+        channelConfigService.updateExpirationSettings(updatedConfig);
+
+        assertChannelLatestUrlEquals(mostRecentItem, true);
+        // TODO: this never gets set because it was inserted as an historical item. We should decrease spoke TTL and wait 'til it passes. like a chump.
```

Comment:
TIME_MACHINE_MODE

Reply:
I think we should put a SPIKE ticket in to look into the time/impact of setting up a hub mode to run with mutable time settings

Reply:
whoops, I forgot to take this todo out. I hadn't rebased onto your fancy new custom release stuff where i could change the ttl...though i think that would fail on CI, since ttl's not going to change on dev.

Reply:
I like the idea of adding a spike for that. I briefly considered the idea of setting up different versions of the hub with stuff already in spoke and/or s3 when the release is installed, but that seemed overly complicated...and again, wouldn't work on our normal dev clusters.